src/core/producer/route/route_producer.cpp

```diff
@@ -53,8 +111,13 @@ class route_producer
     std::shared_ptr<route> route_;
 
     std::optional<std::pair<core::draw_frame, core::draw_frame>> frame_;
+    std::optional<std::pair<core::draw_frame, core::draw_frame>> previous_frame_;
     int                                                          source_channel_;
     int                                                          source_layer_;
+    fix_stream_tag                                               tag_fix_;
+    bool                                                         has_variable_cadence_ = false;
+    bool                                                         is_interlaced_ = false;
```

Comment:
It would be good to prefix these with `source_` to make it easier know what these refer to when reading their usages